Dockerfile

```diff
@@ -62,4 +71,5 @@ COPY changedetection.py /app/changedetection.py
 
 WORKDIR /app
 
-CMD [ "python", "./changedetection.py" , "-d", "/datastore"]
+ENTRYPOINT ["/app/docker-entrypoint.sh"]
+CMD ["python", "./changedetection.py", "-d", "/datastore"]
```

Comment:
You define `/datastore` at line 58 in a env var here and then never use it?

Reply:
Good catch, although `never` is wrong (see entrypoint script). I probably wanted to avoid using a shell, hence no env vars are accessible. Will come up with a solution.